WooCommerce/Classes/POS/Presentation/ProductCardView.swift

```diff
@@ -9,15 +9,21 @@ struct ProductCardView: View {
         self.onProductCardTapped = onProductCardTapped
     }
 
+    var outOfStock: Bool {
+        // TODO: Handle out of stock
+        // wp.me/p91TBi-bcW#comment-12123
+        product.stockQuantity <= 0
+    }
+
     var body: some View {
```

Comment:
Maybe make the whole view a button so that tapping the entire view content adds the product to cart?

```
var body: some View {
        Button(action: {
            onProductCardTapped?()
        }, label: {
            VStack {
                Text(product.name)
                    .foregroundStyle(Color.primaryBackground)
                Text(outOfStock ? "Out of Stock" : product.price)
                    .foregroundStyle(Color.primaryBackground)
                HStack(spacing: 8) {
                    QuantityBadgeView(product.stockQuantity)
                        .frame(width: 50, height: 50)
                    Spacer()
                    Button(action: {
                        onProductCardTapped?()
                    }, label: { })
                    .buttonStyle(POSPlusButtonStyle())
                    .frame(width: 56, height: 56)
                }
            }
            .frame(maxWidth: .infinity)
            .background(outOfStock ? Color.pink : Color.tertiaryBackground)
        })
        .buttonStyle(.plain)
    }
```

Reply:
I'll leave this change out for the moment, since we have a specific button for the `+` and we do not know if tapping in the card itself should have a different behaviour (if any)